### PR TITLE
add daily build test against Node.js nightly build

### LINKED ISSUE #120: Add nightly build to find break earlier.

### PR DESCRIPTION
**Pull request by gengjiawen**
Fix #120

### CODE REVIEW

File: .github/workflows/nodejs.yml [hunk 1]
```diff
@@ -1,29 +1,46 @@
...
+      - run: npm install -g cmake-js@5.3.2
+      - run: npm install
+      - name: Environment Information
+        run: npx envinfo
+      - name: Run Test
+        run: npm test
 
+  nightly-daily-test:
+    runs-on: ubuntu-latest
+    container: gengjiawen/node-build
```
Comment (mhdawson) [inline .github/workflows/nodejs.yml:33]: I wonder if this is something we should try to have as an image managed under the project versus a personal name?

Comment (gengjiawen) [reply .github/workflows/nodejs.yml:33]: My planning is to move this image to `nodejs`, but I guess we can use official one with extra-step.

**Comment by mhdawson** [comment]
@gengjiawen looks like there are some warnings

**Comment by gengjiawen** [comment]
> @gengjiawen looks like there are some warnings

The issue exists in code. It has been exists since we introduced github action.

**Pull request merged**